server/queue/queue-activemq/src/main/java/org/apache/james/queue/activemq/EmbeddedActiveMQ.java

```diff
@@ -49,6 +49,7 @@
         private EmbeddedActiveMQ(FileSystem fileSystem, PersistenceAdapter persistenceAdapter) {
         this.persistenceAdapter = persistenceAdapter;
         try {
+            persistenceAdapter.setDirectory(fileSystem.getFile("file://var/store/activemq/brokers/KahaDB"));
```

Comment:
We don't want, we don't expose ActiveMQ configuration.... 

The other paths are also hard-coded. I don't believe this issue requires this at all.